common/src/main/java/com/wynntils/models/characterstats/CharacterStatsModel.java

```diff
@@ -89,6 +89,9 @@ public double getBlocksAboveGround() {
                         McUtils.player().blockPosition().getZ()))
                 .isAir()) {
             endY--;
+
+            // stop checking beyond the minimum build height as there will never be any blocks below it
+            if (endY < McUtils.mc().level.getMinBuildHeight()) break;
```

Comment:
I think you should return -1 here. Otherwise the function produces an "infinite" value, which is hard to check for with Wynntils functions.
```suggestion
            if (endY < McUtils.mc().level.getMinBuildHeight()) return -1;
```